docs/src/main/sphinx/connector/bigquery.md

```diff
@@ -212,6 +212,9 @@ a few caveats:
   - Enable using Apache Arrow serialization when reading data from BigQuery.
     Read this [section](bigquery-arrow-serialization-support) before using this feature.
   - `true`
+* - `bigquery.arrow-serialization.max-allocation`
+  - The maximum amount of memory the Apache Arrow buffer allocator is allowed to use.
+  - `512MB`
```

Comment:
https://github.com/trinodb/trino/blob/eba7456ddf7373e361880bcda4fb2b22cb3110fd/plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryArrowConfig.java#L24 
The default should be `100MB`